cpu_xenoncat/Linux/asm/equihash_avx1.asm

```diff
@@ -1,7 +1,7 @@
 format elf64
-public EhPrepare
```

Comment:
again about the (de) duplication of platform files.. Can the `Linux/` directory be merged with the "main"? And only use `#ifdef` gates for Win/Linux specific pieces of the code? That will help to ease the development burden and reuse the common parts.